dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -209,8 +201,7 @@ protected PagedDoraWorker(
     mPageSize = mConf.getBytes(PropertyKey.WORKER_PAGE_STORE_PAGE_SIZE);
     mBlockMasterClientPool = blockMasterClientPool;
     mCacheManager = cacheManager;
-    mMetaManager = mResourceCloser.register(
-        new DoraMetaManager(mConf, this, mCacheManager, mUfsManager));
+    mMetaManager = mResourceCloser.register(metaManager);
```

Comment:
Interestingly, I don't have something like `bind(MetaManager).to(DoraMetaManager.class)` in `DoraWorkerModule` and Guice is actually able to notice that and create the `MetaManager` instance for me. Is that okay? I figure we should always call `bind()` explicitly in code?

Reply:
`DoraMetaManager` is not an interface but a concrete class. There is no `MetaManager`.